client/account-recovery/reset-password/reset-password-confirm-form/style.scss

```diff
@@ -0,0 +1,50 @@
+.reset-password-confirm-form__title {
+	font-size: 20px;
+	font-weight: 600;
+	margin-bottom: 16px;
+}
+
+.form-button.reset-password-confirm-form__button {
```

Comment:
Is it necessary the double class declaration? If the second part is specific enough, and it's not reused as it seems, I'd just use `.reset-password-confirm-form__button`.

Same for the others, I won't comment multiple times. ;)